elastic-training/controllers/elastichorovodjob_controller.go

```diff
@@ -34,6 +36,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	aiv1alpha1 "elastictraining/api/v1alpha1"
+
+	sigsv1alpha1 "sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/v1alpha1"
```

Comment:
So it's using the official repo, not your forked repo?

Reply:
Yes, I am using https://pkg.go.dev/sigs.k8s.io/scheduler-plugins now.